docs/source/contributing/developing-core.md

```diff
@@ -171,6 +171,11 @@ In the first session, start the backend.
 make backend-docker-start
 ```
 
+```{note}
+Since `Volto 18.0.0-alpha.42`, the backend data is stored in a `Docker` volume named `volto-backend-data` added within the `Docker` volumes directory when you run `make backend-docker-start` for the first time.
+This way you can easily persist the data between sessions.
+```
+
```

Comment:
```suggestion

When you run this command for the first time, it will download Docker images, configure the backend, create a Docker volume to persist the data named `volto-backend-data`, and start the backend.
You can also delete the volume, and recreate it to start fresh, with the same command.
```

Reply:
@stevepiercy I committed the other changes except this one since I needed to add the info on how to delete the volume check out the notes below. It took so long for me to get the docker volume command to show up as a copy target and the text that follows it as part of the note :( 
It cannot receive any extra ticks to mark it as shell as it messes up the text below it.